lib/polymorphic_integer_type/module_generator.rb

```diff
@@ -0,0 +1,34 @@
+module PolymorphicIntegerType
+  class ModuleGenerator
+    def self.generate_and_include(klass,foreign_type, name)
```

Comment:
Do you have a unit test for this method?

Reply:
It's pretty difficult to unit test this in isolation since it involves setting up polymorphic models and then overriding _some_ of the behaviour(which is basically what we do in the specs already). This functionality was already tested thoroughly in the specs and this just abstracts it away into class